web/libs/editor/src/tags/control/Pairwise.js

```diff
@@ -100,21 +133,22 @@ const Model = types
         InfoModal.error("Incorrect toName parameter on Pairwise, must be two names separated by a comma: name1,name2");
       }
 
-      let selection = {};
-
+      // If selectionstyle is provided, use inline style for backward compatibility.
+      // Otherwise, use Tailwind semantic classes for selection via className.
```

Comment:
That's an interesting approach with tailwind and classnames override! I would simplify it a little: we don't need `selection` var at this level, just assign it directly in both branches to `self._selection`. But it's good anyway